packages/lib/src/vis/shared/VisCanvasProvider.tsx

```diff
@@ -15,13 +15,12 @@ export interface VisCanvasContextValue {
   ordinateConfig: AxisConfig;
   abscissaScale: AxisScale;
   ordinateScale: AxisScale;
-  dataToWorld: (vec: Vector2) => Vector3;
-  worldToData: (vec: Vector3) => Vector2;
+  dataToWorld: (dataPt: Vector2) => Vector3;
+  worldToData: (worldPt: Vector3) => Vector2;
 
   // For internal use only
-  cameraToHtmlMatrix: Matrix4;
-  cameraToHtmlMatrixInverse: Matrix4;
-  cameraToHtml: (vec: Vector3) => Vector2;
+  cameraToHtml: (cameraPt: Vector3) => Vector2;
+  htmlToCamera: (htmlPt: Vector2) => Vector3;
```

Comment:
New internal `htmlToCamera` utility here, which allows me to no longer inject the matrices.